apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -178,6 +178,7 @@ function SelectSchedulePopover(props: { scheduleNames: string[] }) {
                 variant="outlined"
                 onClick={handleClick}
                 sx={{ minWidth, maxWidth, justifyContent: 'space-between' }}
+                disabled={AppStore.getSkeletonMode()}
```

Comment:
Allowing for schedules to be created, edited, and deleted while in skeleton mode creates new bugs and is generally just a bad time.

We don't currently support any aspect of editing schedule names or creating new schedules, so it would be best to disable it when in skeleton mode.